chef_master/source/job_dispatch.rst

```diff
@@ -19,6 +19,18 @@ Chef Automate release 0.6.0 includes a new SSH-based job execution layer. Under
 - View the queue of jobs in the Chef Automate UI
 - View and test runner status via the Chef Automate UI
 
+Known Issues
+--------------
+
+.. warning:: If you need to run windows build and infrastructure nodes, stop now. Job Dispatch is currently incompatible with Windows. If you are running only linux build/infrastructure nodes, please continue.
```

Comment:
This messaging isn't totally correct. You can use windows infrastructure nodes (as long as they can run a push client). The job dispatch feature uses `runners` and those are for linux at the moment (ubuntu, redhat, centos).